EIPS/eip-867.md

```diff
@@ -0,0 +1,177 @@
+
+If accepted, an ERP could easily compile the block at while the changes are to take place, and the source of the actions (which would be the output of the script, in a standardized object output). These can be bundled with the client for seamless execution.
+
+
+## ERP Approval Process
+ERPs are a subclass of EIPs and will, therefore, follow the same approval process.    
+
+### Testing
+*The ERP authors are currently seeking feedback from client teams about the proper testing procedures* 
+
+## Ethereum Client Implementation 
+Clients that choose to adopt the proposal outlined in this EIP will implement a module that scans a designated directory for SCO files (each time the client process launches) to construct a mapping between target blocks and SCO file names.  When starting work on a new block, clients should first consult the set of SCO target blocks discovered and determine if there are any recovery actions required for the current block.  
+
+E.g. in this example, `erpsByTargetBlock` is the mapping between the target block number associated with each ERP's State Change Object and a reference (i.e. file name) to the resource with the actual data.
+
+```
+if (erpsByTargetBlock.get(currentBlock) != null) {
+    try {
+        applyRecoveryActions(erpsByTargetBlock.get(currentBlock));
+     }
+     catch (e) {
+        // recovery actions should be treated as a batch.  
+        // If one fails, they all fail.
+     }
+}
+
+
+// continue with normal block processing...
+```
+
+The `applyRecoveryActions` method must apply the recovery actions in the same order as they are stored in the SCO file.  Clients are responsible for ensuring that no State Change Action will result in an account transferring an amount that is greater than its current balance.  The `toAddress` for both `weiTransfer` and `storeCode` should always be a valid address (i.e. never `0x0`).
+
+Additionally, each block affected by an ERP should include extra data to indicate that the state change occurred.  The extra data included in the block should be the erpId found in the SCO file, converted to hex (i.e. `hexStringToBytes(asciiToHex(erpId))`).  Clients should also validate that the expected header appears in the target block if the block is received from a peer.
+
+The ERP should link to pull requests for each client repository, and these pull requests should link back to the ERP and also contain its EIP preamble and summary. 
+
+Each PR associated with an ERP should consist of a single file (the SCO file) added to the client’s designated SCO directory.  No additional client code should be required.
+
+### Limitations of this EIP
+
+This EIP is focused on standardizing how recovery proposals can be formatted, to optimize readability and eliminate or minimize as much as possible the potential for mistakes or intentional abuses. 
+
+The following are considered out of scope from this EIP:
+- Which fund recovery proposals, if any, should be accepted for implementation. 
```

Comment:
Which fund recovery proposals to accept is the only thing that matters. This EIP does all of the easy work and none of the hard work.

IMHO, this is leveraging the protocol to make it easier to increase the burden on human governance, whereas we should be seeking to minimize this burden.